src/ZoneServer/Pads/Handlers/Swordsman/Barbarian/Barbarian_Pouncing_Abil.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Melia.Shared.Game.Const;
+using Melia.Zone.Network;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.Monsters;
+
+namespace Melia.Zone.Pads.Handlers.Swordsman.Barbarian
+{
+	/// <summary>
+	/// Handler for the Barbarian_Pouncing_Abil pad, creates and disables the effect.
+	/// </summary>
+	[PadHandler(PadName.Barbarian_Pouncing_Abil)]
+	public class Barbarian_Pouncing_Abil : ICreatePadHandler, IDestroyPadHandler, IUpdatePadHandler
+	{
+		/// <summary>
+		/// Called when the pad is created.
+		/// </summary>
+		/// <param name="sender"></param>
+		/// <param name="args"></param>
+		public void Created(object sender, PadTriggerArgs args)
+		{
+			var pad = args.Trigger;
+			var creator = args.Creator;
+
+			Send.ZC_NORMAL.PadUpdate(creator, pad, "Barbarian_Pouncing_Abil", 0, 0, 150, true);
+		}
+
+		/// <summary>
+		/// Called when the pad is destroyed.
+		/// </summary>
+		/// <param name="sender"></param>
+		/// <param name="args"></param>
+		public void Destroyed(object sender, PadTriggerArgs args)
+		{
+			var pad = args.Trigger;
+			var creator = args.Creator;
+
+			Send.ZC_NORMAL.PadUpdate(creator, pad, "Barbarian_Pouncing_Abil", 0, 0, 150, false);
+		}
+
+		/// <summary>
+		/// Called in regular intervals while the pad is on a map.
+		/// </summary>
+		/// <param name="sender"></param>
+		/// <param name="args"></param>
+		public void Updated(object sender, PadTriggerArgs args)
+		{
+			if (!args.Creator.IsBuffActive(BuffId.Pouncing_Buff))
+			{
+				args.Trigger.Destroy();
+				return;
+			}
+
+			args.Trigger.Position = args.Creator.Position;
```

Comment:
Due to the way this particular pad works, it's actually not a big problem here (this pad defines specific slashes that happen at particular times to determine if it hits, so as long as the pad is in the right place when the slashes occur it feels responsive).  The problem is for skills like Rodelero_ShieldCharge and Arditi_Taglio which don't, for them the hitbox is just the player.

Reply:
We can actually time it so it always updates the pad's position before the attack goes off, so it will always be in the correct spot when it needs to be.

Reply:
Am I missing this happening here? I only see the position update in the pad handler. Also keep in mind that updating the position alone would not update the actors inside or raise events. For that, the pad has to receive an actual update.

Reply:
Oh, I'm mistaken, I was thinking of a different skill.  Still, the attack method fires on the pad's update method, which updates the position, so the pad will always be in the right spot when the update goes off, it doesn't use the pad's enter / exit trigger.